FeedStoreChallenge/CoreData/ManagedCache.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Angel Aguilar on 8/15/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+class ManagedCache: NSManagedObject {
```

Comment:
This class is not intended to be inherited thus it should be `final`
Please make the class `final`.

```suggestion
final class ManagedCache: NSManagedObject {
```